who-europe-harvard.csl

```diff
@@ -52,8 +52,8 @@
         <group delimiter=", ">
           <text variable="URL"/>
           <group>
-            <text value="accessed "/>
-            <date variable="accessed ">
+            <text value="accessed"/>
+            <date variable="accessed">
```

Comment:
@zuphilip, shouldn't the `<text/>` element here call the "accessed" term instead of using verbatim text?